### PR TITLE
Update links to default Grafana branch

### PR DESCRIPTION
**Pull request by jdbaldry**
I'll need a review from each CODEOWNER to ensure that this doesn't break anything. I'm hoping that CI will catch if there any places where the links are generated from another source or required in tests for some reason.

If you review on behalf of your squad, and don't want to review all 17 files. Let me know which files you have reviewed so I can ensure that all have been looked at by the appropriate CODEOWNERs.

### CODE REVIEW

File: pkg/plugins/manager/testdata/app-with-child/dist/child/plugin.json
Comment (wbrowne) [inline pkg/plugins/manager/testdata/app-with-child/dist/child/plugin.json:2]: Unfortunately these plugins are signed, so if we update the source we'd need to re-sign them to get a new `MANIFEST.txt`. It's not a big deal to do that, but considering this is test data then it's probably fine to omit.

Comment (jdbaldry) [reply pkg/plugins/manager/testdata/app-with-child/dist/child/plugin.json:2]: Thanks for taking a look!

Given that I wouldn't plan to do the resigning process myself, would you prefer that I revert all the changes to the `pkg/plugins/manager/testdata/` directory?

Comment (wbrowne) [reply pkg/plugins/manager/testdata/app-with-child/dist/child/plugin.json:2]: Yeah I think that's the best option for now 👍

Comment (jdbaldry) [reply pkg/plugins/manager/testdata/app-with-child/dist/child/plugin.json]: Nice! Done in aa323b2e51b

**Review by aangelisc — APPROVED** [review]
Approving this from the Release Guild. The changes are all in old changelog files and tbh most of the links are broken and the files don't exist anymore but I doubt there's much impact due to this given they're archived changelogs.

**Comment by jdbaldry** [comment]
I've not been able to resolve the Swagger error. My machine is generating different output to CI.

If any of the reviewers know what I can do, let me know. If not, I might drop the changes to the Swagger API JSON too.

**Comment by jdbaldry** [comment]
Still need a review from:

- @grafana/backend-platform
- @grafana/grafana-frontend-platform 
- @grafana/plugins-platform-backend 
- @grafana/alerting-backend-product 

Please take a look at your CODEOWNER files if you have some time.

**Review by xnyo — APPROVED** [review]
LGTM!

**Comment by jdbaldry** [comment]
Down to three
- @grafana/backend-platform
- @grafana/grafana-frontend-platform 
- @grafana/alerting-backend-product

**Review by Eve832 — APPROVED** [review]
fwiw, these updates makes sense to me.

**Review by JoaoSilvaGrafana — APPROVED** [review]
👍

**Review by papagian — APPROVED** [review]

**Comment by jdbaldry** [comment]
Only @grafana/alerting-backend-product to go.

Thank you everyone so far for your reviews!

**Comment by jdbaldry** [comment]
I was unable to understand the changes to the OpenAPI spec that resulted from hunks in this patchset so I've discarded them so that this PR can be merged.

**Pull request merged**
